ibc_public/LabelsDb.py

```diff
@@ -0,0 +1,105 @@
+import json
+import sys
+import pandas as pd
+
+import os
+
+
+class LabelsDb:
```

Comment:
I don't see why you need a class here. This makes the design complicated and hard to check/debug.
Please create two functions instead, e.g. `read_labels` and `write_labels`.

Reply:
Done